lib/temporal/worker.rb

```diff
@@ -143,6 +153,10 @@ def shutting_down?
       @shutting_down
     end
 
+    def on_started_test_hook; end
+    def while_stopping_test_hook; end
+    def on_stopped_test_hook; end
```

Comment:
@jeffschoner this is a nitpick but do you mind removing the `test` part of these hook names? So use `on_started_hook`, `while_stopping_hook`, and `on_stopped_hook`?